components/gearScript/gunbag/fn_setGunbagVariableFromArsenalExport.sqf

```diff
@@ -12,6 +12,8 @@ if (typeName _contents isEqualTo "STRING") exitWith
 	[_faction, _unitName, _contents] call f_fnc_setGunbagVariableState;
 };
 
+_contents = _contents#0;
+
```

Comment:
Does this need to have the same conditional check as the change in `components/gearScript/fn_applyLoadoutModifications.sqf`?  If so I can hit that up.

Reply:
probably not since gunbag will only be in use with the new version anyway, for which the gearstrings will be correct. if neede, the same conditional should work